public/assets/js/scripts.js

```diff
@@ -65,16 +92,17 @@ jQuery(document).ready(function() {
 				loadNextStep('#email');
 			},
 			error: function(error){
-				alert("Please enter your registered email ID.");
+				alert('You have not yet been verified by us. Please contact our moderation team.')
 				$('#doctor_email').val('');
```

Comment:
You can remove this if you are doing location.reload().

Reply:
yep, will remove it.